app/controllers/posts_controller.rb

```diff
@@ -1,8 +1,9 @@
 class PostsController < ApplicationController
-  before_action :authenticate_admin!
+  # before_action :authenticate_admin!
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts.includes(:comments)
+
     # @posts = Post.where(user_id: @user)
```

Comment:
- [ ] it would be better to remove the commented code in order to keep the file clean and pro

Reply:
resolve issue